dbt_subprojects/dex/models/_projects/unidex/optimism/unidex_optimism_base_trades.sql

```diff
@@ -0,0 +1,77 @@
+{{ config(
+    alias='base_trades',
+    schema='unidex_optimism',
+    partition_by=['block_month'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key=['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
```

Comment:
this may have been missed in original development, but shouldn't we have `trace_address` as part of unique keys? we should ensure consistency across all models in lineage that feed into aggregator, even if it's a hardcoded dummy value in model